cli/rt/02_console.js

```diff
@@ -400,6 +400,11 @@
     level,
     inspectOptions,
   ) {
+    const proxyDetails = Deno.core.getProxyDetails(value);
+    if (proxyDetails != null) {
+      return inspectProxy(proxyDetails, ctx, level, inspectOptions);
```

Comment:
I think this would become:
```suggestion
      if (inspectOptions.showProxy) {
        return inspectProxy(proxyDetails, ctx, level, inspectOptions);
      }
      return inspectValue(proxyDetails, ctx, level, inspectOptions);
```

Reply:
done!